### DISCUSSION TITLE
SSH-Key

### DISCUSSION THREAD

**Discussion by verenafroelich** [community/community#120989] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Why should this SSH-Key 'id_rsa.pub' be invalid?

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @verenafroelich ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by verenafroelich** [reply]
Many thanks!

**Comment by dane** [comment]
@verenafroelich would you post a screenshot or text showing me where you're seeing that the SSH key is invalid?

**Reply by verenafroelich** [reply]
Thanks to you and to ZFPuhi! I found now what I had to do: I had to add 'cat ~/.ssh/id_ed25519.pub | clip' in my GitBash-Screen, press enter and then a correct key was given! Then I could add this key to GitHub. However: many thanks to you!

**Comment by ZFPuhi** [comment]
Most likely in this case the IdentityFile configuration parameter should be pointed at the private key which the SSH client uses to prove its identity to the remote server. (The remote server, then, should have the contents of id_rsa.pub installed in its authorized_keys file, or an equivalent location).

You should be putting the path to id_rsa, not id_rsa.pub, as an argument to IdentityFile in your ~/.ssh/config . Also double check the permissions of the id_rsa.
